.github/workflows/km-ci-workflow.yaml

```diff
@@ -340,7 +340,7 @@ jobs:
     if: (failure() && github.ref == 'refs/heads/master') ||
       contains(github.workflow, 'noisy')
     # Dependencies. (A skipped dependency is considered satisfied)
-    needs: [km-test, kkm-test, kkm-test-aws, km-test-all]
+    needs: [km-build, kkm-build, km-test, km-test-withpacker, kkm-test, kkm-test-aws, km-test-all]
```

Comment:
Does this job only report fails in the jobs listed here?
It doesn't not follow the dependencies between jobs to decide what to report?

Reply:
I'm not sure about that. It says "A skipped dependency is considered satisfied". I thought this way it is more explicit